src/parser/jobs/mch/modules/Cooldowns.js

```diff
@@ -11,6 +11,7 @@ export default class Cooldowns extends CoreCooldowns {
 		ACTIONS.REASSEMBLE.id,
 		ACTIONS.AUTOMATON_QUEEN.id,
 		ACTIONS.QUEEN_OVERDRIVE.id,
+		ACTIONS.DETONATOR.id,
```

Comment:
I was going to suggest sorting this directly below Wildfire, since the two are tied together. I'm don't feel too strongly either way though, since it should show up on the timeline so infrequently, so ping me once you've either dismissed this comment or made the change and I'll go ahead with the merge.